draft-iab-privacy-partitioning.md

```diff
@@ -623,15 +623,19 @@ If the Oblivious Relay and Gateway collude, they can link Client identity and da
 for each request and response transaction by simply observing requests in transit.
 
 It is not currently possible to guarantee with technical protocol measures that two
-entities are not colluding. However, there are some mitigations that can be applied
+entities are not colluding. Further even if entities do not collude directly, if information
+is revealed to another party, there might be no control about further collusion of data.
+However, there are some mitigations that can be applied
 to reduce the risk of collusion happening in practice:
 
-- Policy and contractual agreements between entities involved in partitioning, to disallow
-logging or sharing of data, or to require auditing.
+- Policy and contractual agreements between entities involved in partitioning to disallow
+logging or sharing of data, as well as careful data minimization or annoymization when auditing is required.
```

Comment:
The auditing being referred to here was auditing that data was not being logged or shared.
```suggestion
- Policy and contractual agreements between entities involved in partitioning to disallow
logging or sharing of data, along with auditing to validate that the policies are being followed.
For cases where some logging is required for service operation and maintenance, policies
should ensure that the data is minimized and anonymized.
```